omod/src/main/java/org/openmrs/module/openconceptlab/web/rest/search/ImportByRunningImportSearchHandler.java

```diff
@@ -35,7 +35,7 @@ public void setImporter(Importer importer) {
     private static final String RUNNING_IMPORT = "runningImport";
 
     private final SearchConfig searchConfig = new SearchConfig("default", RestConstants.VERSION_1 + OpenConceptLabRestController.OPEN_CONCEPT_LAB_REST_NAMESPACE + "/import",
-            Arrays.asList("1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*"),
+            Arrays.asList("1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*"),
```

Comment:
These things are super annoying.  Presumably we should also add in 2.3.x, 2.4.x,... to this and the one below too.  I just added the 2.2.* to get the unit tests to pass in the 2.2 profile, but this will likely cause someone else headaches in the future if we don't update this.

Reply:
Actually, this explains [OCLM-79](https://issues.openmrs.org/browse/OCLM-79).

I'm inclined to replace this with `Collections.singletonList("1.8.* - 2.*")` and similarly elsewhere.

Reply:
Done.  Replaced in these 2 classes.  Thanks for pointing out that you can do that.